pkg/mtu/detect_linux.go

```diff
@@ -40,7 +39,7 @@ func getRoute(externalProbe string) ([]netlink.Route, error) {
 
 	routes, err := netlink.RouteGet(ip)
 	if err != nil {
-		return nil, fmt.Errorf("unable to lookup route to %s: %s", externalProbe, err)
+		return nil, fmt.Errorf("unable to lookup route to %s: %w", externalProbe, err)
```

Comment:
this is interesting change. Just curious if there is any plan to update fmt.Errorf("....%s") to fmt.Errorf("....%w") in the code base.

Reply:
That's a good point but it should probably be something for another PR. A quick grep through the codebase gives me ~500 occurrences of this pattern.